winui/Pyramid-Chart/Selection.md

```diff
@@ -97,48 +97,6 @@ this.Content = chart;
 
 ![Multi selection support in WinUI Chart](Selection_images/WinUI_chart_multi_selection.png)
 
-## Changing Cursor while Selection
-
-[Cursor](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionBehavior.html#Syncfusion_UI_Xaml_Charts_ChartSelectionBehavior_Cursor) property allows to define the cursor when mouse is hovered over the segment with segment selection enabled.
-
-{% tabs %}
-
-{% highlight xml %}
-
-<<chart:SfPyramidChart x:Name="chart"  
-                    SelectionBrush="Red"
-                    Palette="BlueChrome"  
-                    ItemsSource="{Binding Data}" 
-                    XBindingPath="Category"
-                    YBindingPath="Value">
-
-        <chart:SfPyramidChart.Behaviors>
-            <chart:ChartSelectionBehavior Cursor="Hand" />
-        </chart:SfPyramidChart.Behaviors>
-
-</chart:SfPyramidChart>
-
-{% endhighlight %}
-
-{% highlight c# %}
-
-SfPyramidChart chart = new SfPyramidChart();
-chart.SetBinding(SfPyramidChart.ItemsSourceProperty, new Binding() { Path = new PropertyPath("Data") });
-chart.XBindingPath = "Category";
-chart.YBindingPath = "Value";
-chart.SelectionBrush = new SolidColorBrush(Colors.Red);
-ChartSelectionBehavior selection = new ChartSelectionBehavior();
-selection.Cursor = Windows.UI.Core.CoreCursorType.Hand;
-chart.Behaviors.Add(selection);
-. . .
-this.Content = chart;
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![Changing cursor while selection support in WinUI Chart](Selection_images/WinUI_chart_cursor.png)
-
 ## Events
 
 The following selection events are available in [SfPyramidChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfPyramidChart.html).
```

Comment:
@naveenkumar-sanjeevirayan : Currently selection events moved to selection behavior. Change sentence in all charts.

Reply:
Updated changes in all charts.